### ISSUE TITLE
Feature request: use with Reapeater

### ISSUE THREAD

**Issue by terreb** [TheOriginalJosh/nativescript-slides#60]
Would be awesome if sildes could be added dynamically with Repeater.

**Comment by sm1ddy** [comment]
Yeah I was thinking the same thing, some functionality like how listviews are dynamically populated from a collection would be great,

**Comment by terreb** [comment]
@sm1ddy, have a look at [nativescript-carousel](https://github.com/manijak/nativescript-carousel).
